WooCommerce/Classes/ViewRelated/Dashboard/Settings/POS/POSEligibilityChecker.swift

```diff
@@ -25,7 +26,7 @@ final class POSEligibilityChecker: POSEligibilityCheckerProtocol {
                 .eraseToAnyPublisher()
         }
 
-        return Publishers.CombineLatest(isWooCommerceVersionSupported, isPointOfSaleFeatureFlagEnabled)
+        return Publishers.CombineLatest(isWooCommerceVersionSupportedAndFeatureSwitchEnabled, isPointOfSaleFeatureFlagEnabled)
```

Comment:
It's probably time to remove `isPointOfSaleFeatureFlagEnabled` as well.

The eligibility criteria should be:
- isWooCommerceVersionSupported
- isFeatureSwitchEnabled

Reply:
Right now, `isPointOfSaleFeatureFlagEnabled` is still checking the remote feature flag https://github.com/woocommerce/woocommerce-ios/blob/919db658149af1606cdbb8d54d8c26e901ce6190/WooCommerce/Classes/ViewRelated/Dashboard/Settings/POS/POSEligibilityChecker.swift#L94-L104. Have we decided to remove the remote POS feature flag in WPCOM?

Reply:
Good point. Not yet, we can keep this for safety for now in case we need to disable it.